thoth/storages/graph/postgres.py

```diff
@@ -4822,6 +4894,47 @@ def sync_revsolver_result(self, document: Dict[str, Any]) -> None:
                     platform=entry["platform"],
                 )
 
+    def _check_package_solved(self, session: Session, package_name: str, package_version: str,
```

Comment:
This method should return package_version_entiy ID and package_version ID required below. If those values are not returned the method will fail, which is what we want. If some package is not solved first, we do not want to sync anything else in the database for that package.

Reply:
> This method should return package_version_entiy ID and package_version ID required below in the SIAggregated sync. If those values are not returned the method will fail, which is what we want. If some package is not solved first, we do not want to sync anything else in the database for that package.

Reply:
https://github.com/thoth-station/storages/pull/1963/commits/f409f97503d2f429e6b55ba3f072dc932ae4393e
Something like this?

Reply:
Yes but realized we do not need solver info here. See comments below

Reply:
We just need to know if the package version index exist in the database in PPV table, which means it is solved. Otherwise sync fails.